tools/local-actions/changelog/lib/main.ts

```diff
@@ -46,9 +47,11 @@ async function run(): Promise<void> {
   AuthenticatedGitClient.configure(await getAuthTokenFor(ANGULAR_ROBOT));
   /** The authenticed GitClient. */
   const git = AuthenticatedGitClient.get();
-  git.run(['config', 'user.email', '56403804+angular-robot[bot]@users.noreply.github.com']);
+  git.run(['config', 'user.email', 'angular-robot@google.com']);
   git.run(['config', 'user.name', 'Angular Robot']);
 
+  /** The url of the git repository with the authenticated token set. */
+  const authenticatedGitUrl = addTokenToGitHttpsUrl(git.getRepoGitUrl(), git.githubToken);
```

Comment:
The `git.getRepoGitUrl` method from the authenticated client already incorporates the token? is that really needed?

Reply:
Correct, I don't know why when I was testing before that it didn't include it.